src/osemgrep/cli_scan/Rule_fetching.ml

```diff
@@ -110,6 +110,9 @@ let import_callback base str =
               *)
              Logs.debug (fun m ->
                  m "trying to download from %s" (Uri.to_string url));
+             (* TODO: ask for JSON in headers which improves performance
```

Comment:
I wonder how much the difference is in OCaml. The downside of JSON is that we can't report nice error spans for rules. We can't already, but it would be cool if after the osemgrep port we could